client/src/pages/UserLogin/components/LogInContainer.js

```diff
@@ -74,36 +74,39 @@ const LogInContainer = (props) => {
               {loginText}
             </Button>
           </Grid>
-          <Grid item container justifyContent="center" md={1}>
-            <Typography variant="subtitle2" sx={whiteText}>
-              Login to access site
-            </Typography>
-            <Button
-              color="secondary"
-              variant="contained"
-              size="small"
-              type="button"
-              sx={buttonTextSize}
-              onClick={handleChange}
-            >
-              Log In
-            </Button>
-          </Grid>
-          <Grid item container justifyContent="center" md={1}>
-            <Typography variant="subtitle2" sx={whiteText}>
-              Don't have an account?
-            </Typography>
-            <Button
-              color="secondary"
-              variant="contained"
-              type="button"
-              size="small"
-              sx={buttonTextSize}
-              onClick={handleChange}
-            >
-              Sign Up
-            </Button>
-          </Grid>
+          {!isLoggingIn ? (
+            <Grid item container justifyContent="center" md={2}>
+              <Typography variant="subtitle2" sx={whiteText}>
+                Login to access site
+              </Typography>
+              <Button
+                color="secondary"
+                variant="contained"
+                size="small"
+                type="button"
+                sx={buttonTextSize}
+                onClick={handleChange}
+              >
+                Log In
+              </Button>
+            </Grid>
```

Comment:
don't repeat yourself principle bai, gibalik ra nimo basically ang component pero different texts lang, how about just create these texts as a variable nya didto lang i perform ang logic
ex:
`footerText = isLoggingIn? "Don't have an account?" : "Login to access site"` 

nya same lang for the button maybe `footerButton` or kanang better name if maka think ka

Reply:
resolved